python_course_1604/class_02/exercize_04_iterators.py

```diff
@@ -24,9 +24,17 @@ def execute_commands(commands):
     while True:
         # HINT: look how next function behave when sequence terminates
 
-        command = next(iterator)
-
-        if command == 'ping':
-            print('pong')
-        else:
-            print('error')
+        try:
+            command = next(iterator)
+
+            if command == 'quit':
+                break
+            elif command == 'skip':
```

Comment:
You missed 100% of coverage because you implemented an unrequited command: skip. Please note the specs for this exercise are not the same as for the previews one.